modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -14,11 +14,16 @@ In the table, features are marked with the following statuses:
 * *GA*: _General Availability_
 * *NA*: _Not Applicable_
 
+[IMPORTANT]
+====
+Before upgrading to {product-title} 4.13, move out of the `stable` channel.
```

Comment:
The expression "move out" doesn't seem to fit here and isn't used elsewhere in the openshift-docs repo. Is there another more common way of expressing the same idea?

Reply:
@rolfedh Addressed. Thank you!